ristonrodrigues723.md

```diff
@@ -0,0 +1,172 @@
+ 
+ i will be refrancing the following builds-
+ 1)  https://www.instructables.com/Free-Smart-Home/
+ 2)  https://www.instructables.com/Raspberry-Pi-Home-Automation-Control-lights-comput/
+ 3)  https://www.instructables.com/Home-Automation-4-5-6/
+ 4)  https://www.instructables.com/Smart-Home-by-Raspberry-Pi/
+ 5)  https://www.instructables.com/DIY-Cheap-Safety-Alarm-System-W-Raspberry-a-Smarte/
+ 
+the following parts will be used again and again: throughout the building procdess:
+a)|Plusivo Soldering Kit with Multimeter (EU Plug) | - soldering the wires into boards , the sensors to esp 32 modules and sensors together also will be used to do the process of removing soldring
+b)| Orange Raspberry Pi 4 Advance Kit   |  mainly used in the central automation hub but can also be used for other modules.
+c)| 20 CM 40 Pin Dupont Cable Male/Male, Male/Female, Female/Female Cable Combo |- the wires to be used theoughout the module
+d)| Tactile Push Button Switch set | buttons to be used in the esp 32 module enabled circuts
+e)| GLUN 11 mm Transparent HOT MELT glue sticks - to be used in the hot glue gun.
+f) cardboard and plywood sheets/pieces- already owned:- creating containers for the modules and the wooden bin in petbfood module.
+g)| ApTechDeals 60W /100 Watt Dual Watt Glue Gun |- glueing together the wooden and cardboatrd pieces and cases.
+i want to  develop different modules such as:
+
+1)Control camera and security module- will be used to develope a camara module which can be put on the house gate for security gate 
+
+will make use of one of: 
+- 1 x| D10 3.7V Rechargeable Lithium ion Battery Power Bank Cell , these cells will also be used on the modules that will be placed outside,
+- 1 x| ESP32-CAM-MB MICRO USB Download Module for ESP32 CAM Development Board|-the main camara to be connected to it,
+- 1 x| I2C ADS1115 16 Bit ADC 4 channel Module with Programmable Gain Amplifier | will be used to check status of camera and the gate ,
+- 1 x | MC-38 Wired Door Window Sensor Magnetic Switch Home Alarm System |, will be connected to the esp 32 board will be placed next to the gate bto check opening closing status.
+
+
+
+2)Module Outside Station- ie. unit for water tank,  weather level and to check its lid status
+
+The purpose of the outside station is designed to measure the humidity of the air, the temperature and the level of your water tank. Each variable is sent to the website.
+
+will use the following:
+- 1 x | ESP32-CAM-MB MICRO USB Download Module for ESP32 CAM Development Board|
+- 1 x Battery  | D10 3.7V Rechargeable Lithium ion Battery Power Bank Cell
+- 1 DHT22 : Sensor for humidity and temperature 
+- 1 x | HC-SR04-Ultrasonic Range Finder |: Ultrasonic sensor for the level of the watertank (
+- Some “Dupont” wires x | 	Female to Female DuPont Line 40 Pin 30cm |- to connect the components together
+- 1 x |37-1K - Resistance:1kohm |- will be conne cted in the module for resistenc.
```

Comment:
why is this resistor a 1k ohm instead of 10k ohm?
<img width="794" alt="Screenshot 2023-01-13 at 11 53 00 AM" src="https://user-images.githubusercontent.com/65808924/212377018-8db41052-e569-492a-8dfe-3fef3ca979f9.png">

Reply:
oh resistors arent much costly i may have made a mistake ill  change it to 10 k


> hey @ristonrodrigues723 thanks for pushing the changes! i noticed that some of the parts listed are different from the smart home tutorial you listed, did you intentionally modify it? (100% ok just checking if you have the parts you need) thanks!
> 
> on your friend's thing, that sounds cool, looking forward to a PR :)

hey, sis if my friend submits a plan tomorrow will it pass the review? there would only be one day remaining and he would need to make the improvements suggested, he devised a detailed plan and formatted it as per the guidelines but didn't save the file, he was directly adding in GitHub and his laptop crashed and everything kinda just disappeared.

Reply:
i didnt see it properly and serched for 1 k one, thanks for pointing it out.

Reply:
@ristonrodrigues723 any PRs that come in before Jan 15 11:59pm ET will be reviewed hehe :) thank you! and, cool, can you please change it in your budget too?